.github/ISSUE_TEMPLATE/server-migration.md

```diff
@@ -0,0 +1,97 @@
+---
+name: Server migration
+about: Checklist to migrate to an upgraded server
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Checklist based on general guide https://github.com/openfoodfoundation/ofn-install/wiki/Migrating-a-Production-Server
+
+## 1. Setting up the new server
+- [ ] Check old server config for any additional services to be aware of (eg `/etc/nginx/sites-available`). Document any necessary steps for migration.
+- [ ] Hosting: provision new server with Ubuntu 20
+- [ ] DNS: add temporary domain (eg `prod2.openfoodnetwork.org`)
+
+### config
+- [ ] Add temporary name to `inventory/hosts`
+- [ ] Review `host_vars/x/config.yml`, clean up if needed
+  - Make a copy for the temp hostname, add temp domain to bottom of `certbot_domains`
+- [ ] Review `ofn-secrets:x_prod/secrets.yml`, clean up if needed
+   - Change to shared bugsnag projects
+   - Don't bother making a copy of this one
+
+### setup
+Ensure you have the correct secrets.
+`ansible-playbook -l x_prod2 -e "@../ofn-secrets/x_prod/secrets.yml playbooks/`
+- [ ] `letsencrypt_proxy.yml`
+- [ ] `setup.yml`
+- [ ] `provision.yml`
+- [ ] `deploy.yml`
+- [ ] `db_integrations` (Permit DB access for n8n, Metabase)
+- [ ] Ensure sidekiq is disabled, to avoid creating subscription orders: `systemctl disable sidekiq`
+
+### initial migration
+`ansible-playbook -l x_prod -e rsync_to=x_prod2 playbooks/`
+- [ ] Setup direct ssh access for `ofn-admin` and `openfoodnetwork` as per guide
+- [ ] `db_transfer.yml`
+- [ ] `transfer_assets.yml`
+
+## 2. Testing
+ - [ ] test `reboot`
+ - [ ] send test mail (`/admin/mail_methods/edit`). 
+ - [ ] terms of service file: `/admin/terms_of_service_files`
+ - [ ] shop catalogue display correctly, with images, add to cart, begin checkout, login
+  - note: check cookies if login won't work
+ - Check integrations 
+   - [ ] Payments (check Stripe connect status `/admin/stripe_connect_settings/edit`)
+   - [ ] New Relic
+   - [ ] Bugsnag
+
+## 3. Migration
+### preparation
+- [ ] `deploy.yml -l x_prod2 -e "git_version=vX.Y.Z"` matching version with current prod
```

Comment:
I would make sure to have an empty database before deploying anything on the new server. A deployment is starting Sidekiq and may have other side effects in migrations we don't know yet.

Reply:
But if sidekiq is `disabled`, it wouldn't get started would it? Hmm unless ansible automatically enables it for you.

The database will be dropped by the subsequent `db_transfer.yml` anyway. So I don't see the need to empty the database, except to perhaps save time if there are any migrations. But if you think we'll avoid unknown side-effects, that's a good enough reason to do it.